test/integration/security-rules.spec.ts

```diff
@@ -185,7 +211,7 @@ describe('admin.securityRules', () => {
       return admin.securityRules().releaseStorageRuleset(oldRuleset);
     }
 
-    after(() => {
+    afterEach(() => {
```

Comment:
Rename this to `revertStorageRulesetIfModified`

Reply:
Done.